code/modules/projectiles/radbolt.dm

```diff
@@ -34,5 +34,9 @@ toxic - poisons
 	hit_ground_chance = 50
 	//Can we pass windows
 	window_pass = 1
-	//no visible message upon bullet_act
+	//no visible message upon bullet_act and no armor block message
 	silentshot = 1
+	//Bullet equivalent impact_range
+	impact_range = 0
+	//No armor block messages
```

Comment:
these comments conflict

Reply:
They have some overlapping functionality in this line which I believe means silentshot causes no armor block messages.

https://github.com/goonstation/goonstation/blob/03327ced6d8e1bff6c7eca6dd0669645ff00bdbd/code/mob/living.dm#L1876

Reply:
As far as I can see, datum/projectile variable nomsg duplicates silentshot functionality and doesn't actually do anything that silentshot doesn't other than preventing building an armor_msg string... which is blocked if silentshot is set anyway.

Based on that nomsg seems to be an unnecessary variable on the datum.

Reply:
best to remove then

Reply:
yeah nomsg can be removed entirely and replaced by silentshot where needed